static/js/ui.js

```diff
@@ -42,7 +42,18 @@ exports.report_message = function (response, status_box, cls, type) {
               .text(response).stop(true).fadeTo(0, 1);
     }
 
-    status_box.show();
+    status_box.addClass("show");
+};
+
+exports.hide_error = function ($target) {
```

Comment:
Just an FYI, we are really trying to avoid adding code to `ui.js`, to cut down on our circular dependencies.   I don't think you need to worry about it for this change, but we eventually want to extract a module called `ui_report.js` for these types of things.